src/main/java/com/groupbyinc/api/Query.java

```diff
@@ -1193,4 +1186,72 @@ public Query addQueryUrlParams(String key, String value) {
         this.queryUrlParams.put(key, value);
         return this;
     }
+
+    /**
+     * <code>
+     * Add a biasing profile, which is defined at query time. This allows you to override or augment
+     * biasing profiles defined in Command Center. By default, biases defined here will augment the ones
+     * defined in Command Center.
```

Comment:
Lets add: 

`Set override to true to only use the biasing profile defined here and ignore the ones created in command center.`